### DISCUSSION TITLE
MFXFilteredComboBox: changing default prompt text

### DISCUSSION THREAD

**Discussion by FelipeAumannRS** [palexdev/MaterialFX#337] [Q&A]
Do anybody know how to change the filtered ComboBox prompt text (which is "Search...")?

In the mfx sample application I noticed that this text is actually a I18n property and it's added to the filtered combo box search text field this way:

`MFXTextField searchField = new MFXTextField("", I18N.getOrDefault("filterCombo.search")) {`

Unfortunately I have no idea how to access this textField in order to change the promp text, that's the goal

**Comment by infinite-dev22** [comment]
The ```MFXFilterComboBox``` search prompt text is set within it's skin, that's where ```MFXTextField searchField = new MFXTextField("", I18N.getOrDefault("filterCombo.search"));``` is called, set or refrenced. For now I don't think setting this prompt text to a custom one is currently supported. The only way you may achieve this is by creating yourself a custom skin, add all your features as required and setting it onto your ```MFXFliterComboBox```. Basically just copy the existing skin, modify it to custom and set it as the new skin for ```MFXFliterComboBox``` in your program.
